.github/workflows/README.md

```diff
@@ -99,14 +99,55 @@ graph TD
 - Pushes HTML to gh-pages branch, which is linked to https://docs.opendp.org
 
 
-## Making one-off releases
+## Knobs on the Release Workflow
 
 One-off releases can be made with the
 [`release.yml` workflow](https://github.com/opendp/opendp/actions/workflows/release.yml)
 on github, or with the `gh` command line tool. Parameters:
 
 - **Target channel** controls how the release is tagged, and what semantic version is given to the release. There is a git branch with the same name for each channel.
-- The **sync the Channel from upstream?** checkbox is for when you want to update the `nightly`, `beta` or `stable` branches.
+- The **sync the Channel from upstream?** checkbox is for when you want to update the `nightly`, `beta` `stable` or `dev` branches. This replaces the contents of the respective branch with the branch the workflow is run from ("Use workflow from"). In practice, you'll almost always enable this.
 - Update the **version counter** accordingly when you want to release multiple nightlies or betas in the same day.
 - **Dry runs** get sent to test-pypi, and don't update the docs
 - **Fake** is for developer convenience when debugging CI: it skips compilation and inserts dummy binaries instead
+
+## Making a release
+
+1. Ensure that changelog is up-to-date and version in `main` matches the version you want to release (otherwise see step 3).
+
+2. We use a release train with nightly, beta and stable channels. Repeatedly run the Release workflow through each channel (see [Making one-off releases](#Making-one-off-releases)).
```

Comment:
```suggestion
2. We use a release train with nightly, beta and stable channels. Repeatedly run the Release workflow through each channel (see preceding section).
```

I find really short hop links a little confusing. If you do want to keep it, update the link to reflect the current title.